src/platform/src/main/java/org/geoserver/platform/resource/Paths.java

```diff
@@ -459,12 +495,60 @@ public static String convert(String path, String filename) {
         return toPath(STRICT_PATH, resolvedPath);
     }
 
+    /**
+     * Pattern used to recognize absolute path in Windows, as indicated by driver letter reference
+     * (included {@code :}), followed by as slash character.
+     *
+     * <p>Aside: A drive letter reference on its own results in a relative path (relative to the
+     * current directory for that drive).
+     */
+    static final Pattern WINDOWS_DRIVE_LETTER = Pattern.compile("^\\w\\:/.*$");
```

Comment:
Hum... I think that now I can enter a path with backward slashes in the UI and it will work. 

I read somewhere in your comments (and hopefully I just misunderstood) that we can have our own conventions for paths... that would be wrong IMHO, the paths are visible to users, they enter them also manually, so we should be respecting the conventions of the OS they are already using.

Reply:
I added some docs to the PR to try and make it more clear